### ISSUE TITLE
results on pancreas dataset

### ISSUE THREAD

**Issue by Huiimin5** [HiLab-git/DTC#10]
Hi thanks a lot for releasing the code. On pancreas dataset, I have followed the same pipeline of preprocessing, including resampling, center crop and normalization but I can only get 40.86 dice trained with 12 labeled data and 60.43 dice trained with all training data. Could you please specify what augmentation you used in your paper? because after I remove rotation augmentation and use random crop only as on-the-fly augmentation, I can get a similar results as yours.

**Comment by Luoxd1996** [comment]
Hi, 
Yeah, there are some tricks, you have got the most important one, remove the random rotate/flip for training or rotate at a very small angle [-15,15].
Best,
Xiangde.

**Comment by Huiimin5** [comment]
Thank you so much for your response. I have several more details to check with you. 
1. Did you center crop first, normalize second or normalize first center crop second as in UA-MT? 
2. In the resampling of raw CT image and ground truth, what order did you use for interpolation, respectively? 
3. Did you center crop over 3 dimensions or only two (x and y), as in UA-MT?

**Comment by Luoxd1996** [comment]
Hi,
More details you can find in our paper.
Best,
Xiangde.

**Comment by nguyenpbui** [comment]
Dear authors,
Firstly, thank you for your amazing work.
I tried to reproduce your paper on Pancreas dataset. I've read your paper and implemented very carefulling including pre-processing data and data augmentation on-the-fly but the results are quite low compared to yours. Specifically, the results are **39.50 Dice, 26.83 Jaccard, 15.24 ASD, and 34.70 95HD** while your results are **70.63 Dice, 56.72 Jaccard, 6.29 ASD, and 22.54 95HD**
Can you explain about it or update your repo with training and testing code for Pancreas dataset?
Thank you

**Comment by Luoxd1996** [comment]
Hi，
You can turn off the random rotation, it can not bring any performance gain.
Best,
Xiangde.

nguyenpbui ***@***.***> 于2022年1月12日周三 17:36写道：

**Comment by nguyenpbui** [comment]
Hi,
I just turned off the random rotation but the results are not improved.  Can you provide the training settings for Pancreas data set?
Thanks

**Comment by nguyenpbui** [comment]
Hi @Huiimin5 , how did you set the similar results on training VNet with 12 labeled scans with the paper? Can you share your implementation or training settings?
Thank you

**Comment by daobinhuang** [comment]
@Huiimin5  you got similar results on pancreas dataset,   can you share your code ?  thanks
